lib/std/Build.zig

```diff
@@ -96,6 +96,10 @@ cache_root: Cache.Directory,
 global_cache_root: Cache.Directory,
 cache: *Cache,
 zig_lib_dir: ?LazyPath,
+// The hash of these bytes will be used as an init value for the build prng.
+// It allows receiving any input as seed, and printing it to output for
+// reproducibility.
+seed: []const u8,
```

Comment:
if you want to expose this to build scripts, let's make that a separate proposal.